Makefile

```diff
@@ -327,6 +331,30 @@ test-env-info:
 
 # ============================================================================
 
+# PA11Y TASKS
+
+PA11Y_DIR = pa11y
+PA11Y_CONF = $(PA11Y_DIR)/.pa11yci
+PA11Y_REPORT_JSON = $(PA11Y_DIR)/pa11y-ci-results.json
+
+.PHONY: test-pa11y
+test-pa11y:
+	docker-compose exec php-fpm sh -c "./node_modules/pa11y-ci/bin/pa11y-ci.js -c $(PA11Y_CONF) -j -T 1000 > $(PA11Y_REPORT_JSON)"
```

Comment:
Is `-T 1000` a threshold option? Because we have [errors](https://9211-80401279-gh.circle-artifacts.com/0/planet4-docker-compose/artifacts/pa11y/index.html), but the job passed

Reply:
It is a threshold, I moved it to the CI command `test-pa11y-ci` only, the CI then [decides on how to fail](https://github.com/greenpeace/planet4-master-theme/pull/1165/files#diff-1d37e48f9ceff6d8030570cd36286a61R185) based on the presence of errors in the report. For the local command `test-pa11y`, it includes errors only (no warning or notice) and fails accordingly.